ai_ta_backend/service/export_service.py

```diff
@@ -35,15 +35,16 @@ def export_documents_json(self, course_name: str, from_date='', to_date=''):
 
     response = self.sql.getDocumentsBetweenDates(course_name, from_date, to_date, 'documents')
     # add a condition to route to direct download or s3 download
-    if response.count > 1000:
+    if response.count > 500:
       # call background task to upload to s3
 
       filename = course_name + '_' + str(uuid.uuid4()) + '_documents.zip'
       s3_filepath = f"courses/{course_name}/{filename}"
       # background task of downloading data - map it with above ID
       executor = ProcessPoolExecutor()
       executor.submit(self.export_data_in_bg, response, "documents", course_name, s3_filepath)
-      return {"response": 'Download from S3', "s3_path": s3_filepath}
```

Comment:
Let's keep the response as `Download from S3`. The user message will be created on frontend on the basis of this response.

Reply:
reverted!